hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/test/java/org/hypertrace/traceenricher/enrichment/enrichers/ApiBoundaryTypeAttributeEnricherTest.java

```diff
@@ -300,6 +301,34 @@ public void testEnrichEventWithGrpcNoAuthority() {
     Assertions.assertEquals(EnrichedSpanUtils.getHostHeader(innerEntrySpan), "testHost");
   }
 
+  @Test
+  public void testEnrichEventWithGrpcNoAuthorityButRequestMetadataHost() {
+    mockProtocol(innerEntrySpan, Protocol.PROTOCOL_GRPC);
+    org.hypertrace.core.datamodel.eventfields.grpc.Grpc grpc =
```

Comment:
Can you remove lines from 307 to 315 that are not required?
Below code,

```
 org.hypertrace.core.datamodel.eventfields.grpc.Grpc grpc =
        mock(org.hypertrace.core.datamodel.eventfields.grpc.Grpc.class);
    when(innerEntrySpan.getGrpc()).thenReturn(grpc);
    Request request = mock(Request.class);
    when(grpc.getRequest()).thenReturn(request);
    RequestMetadata metadata = mock(RequestMetadata.class);
    when(request.getRequestMetadata()).thenReturn(metadata);
    when(metadata.getAuthority()).thenReturn("localhost:443");
```

Reply:
resolved